swap/src/protocol/alice/steps.rs

```diff
@@ -35,11 +34,11 @@ pub async fn negotiate(
     xmr_amount: monero::Amount,
     event_loop_handle: &mut EventLoopHandle,
     config: Config,
-) -> Result<(ResponseChannel<AliceToBob>, alice::State3)> {
+) -> Result<(PeerId, alice::State3)> {
     trace!("Starting negotiate");
 
     // todo: we can move this out, we dont need to timeout here
```

Comment:
🙃 Is the `ToDo` still valid? Could we action it as part of this PR?

Reply:
I tried to action it as part of #157. It will be taken care of when starting to build nectar.